packages/components/src/tab-panel/index.js

```diff
@@ -29,87 +29,71 @@ const TabButton = ( { tabId, onClick, children, selected, ...rest } ) => (
 	</Button>
 );
 
-class TabPanel extends Component {
-	constructor() {
-		super( ...arguments );
-		const { tabs, initialTabName } = this.props;
+const TabPanel = ( {
```

Comment:
You should use named functions for better debug stack traces. In other words, use `function TabPanel(...) {...}` instead of `const TabPanel = (...) => {...};`

Also, in this particular case, since this is the default export, you can just do the default export here (rather than at the bottom of the file), e.g.:

`export default function TabPanel(...`